src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -585,3 +596,241 @@ describe('VAOS <RequestedAppointmentDetailsPage>', () => {
     expect(screen.baseElement).not.to.contain.text('Canceled');
   });
 });
+
+describe('VAOS <RequestedAppointmentDetailsPage> with VAOS service', () => {
```

Comment:
I think we also need a test for the requested appointments list page, to cover the new call we added there.